lib/iex/lib/iex/autocomplete.ex

```diff
@@ -309,23 +309,24 @@ defmodule IEx.Autocomplete do
   end
 
   defp match_module_funs(funs, hint) do
-    for({fun, arity} <- funs, name = Atom.to_string(fun), String.starts_with?(name, hint), do: %{
-      kind: :function,
-      name: name,
-      arity: arity
-    })
-    |> Enum.sort_by(&{&1.name, &1.arity})
+    for {fun, arity} <- funs,
+        name = Atom.to_string(fun),
+        String.starts_with?(name, hint),
+        do: %{
+              kind: :function,
+              name: name,
+              arity: arity
+            }
+            |> Enum.sort_by(&{&1.name, &1.arity})
```

Comment:
Sorry for the continued requests. Do you think it would look better as a proper `do ... end` block? And the one below?

Reply:
I have actually tried a variant of that as well. Not sure which output is preferrable. Submit another commit following this.